spark/dl/src/main/resources/types

```diff
@@ -0,0 +1,21 @@
+textType, textLabel
+alt.atheism, 1
+comp.graphics, 2
+comp.os.ms-windows.misc, 3
+comp.sys.ibm.pc.hardware, 4
+comp.sys.mac.hardware, 5
+comp.windows.x, 6
+misc.forsale, 7
+rec.autos, 8
+rec.motorcycles, 9
+rec.sport.baseball, 10
+rec.sport.hockey, 11
+sci.crypt, 12
+sci.electronics, 13
+sci.med, 14
+sci.space, 15
+soc.religion.christian, 16
+talk.politics.guns, 17
+talk.politics.mideast, 18
+talk.politics.misc, 19
+talk.religion.misc, 20
```

Comment:
we should put it somewhere else; if not, we should put it under a specific subfolder (or at least rename it per the specific example name).